.mailmap

```diff
@@ -264,6 +267,8 @@ Pauline Barmby               <pbarmby@uwo.ca> <pbarmby@uwo.ca>
 Perry Greenfield             <perry@stsci.edu>
 P. L. Lim                    <2090236+pllim@users.noreply.github.com>
 P. L. Lim                    <2090236+pllim@users.noreply.github.com> <lim@stsci.edu>
+Prajwel Joseph               <prajwel.pj@gmail.com>
+Prajwel Joseph               <prajwel.pj@gmail.com> Prajwel Joseph <prajwel.pj@gmail.com>
```

Comment:
I don't get this one. Aren't the two names/emails exactly the same? Am I missing something here?